sdk/samples/core/mosquitto_rpc_server_sample.c

```diff
@@ -105,7 +104,7 @@ static void timer_callback(union sigval sv)
   }
 
   pending_command.content_type = AZ_SPAN_FROM_BUFFER(content_type_buffer);
-  pending_command.response_topic = AZ_SPAN_FROM_BUFFER(response_topic_buffer);
+  pending_command.request_topic = AZ_SPAN_FROM_BUFFER(request_topic_buffer);
```

Comment:
Note: (didn't check - we might already do this) - it would be preferable if we could verify the size of the buffer in the `_init()` API (and return `AZ_ERROR_NOT_ENOUGH_SPACE` if these buffers are too small).

I was also thinking that now, with multiple buffers, detection of which buffer is too small could get complicated. We might want to have some preconditions when there isn't sufficient space, so that the app blocks on the offending line / buffer.

Reply:
Will investigate this once I have the public API